src/api/libcellml/parser.h

```diff
@@ -0,0 +1,168 @@
+/*
+Copyright 2015 University of Auckland
```

Comment:
Nitpicking (?): as far as I know, there is still no final decision that has been made regarding the copyright statement that we ought to use, but here we use 2015 while for the `Logger` class we use 2016...?